config/packages/security.yaml

```diff
@@ -5,14 +5,40 @@ security:
     # https://symfony.com/doc/current/security.html#loading-the-user-the-user-provider
     providers:
         users_in_memory: { memory: null }
+        app_user_provider:
+            entity:
+                class: App\Entity\Contact
+                property: email
+
+    role_hierarchy:
+        ROLE_ADMIN: ROLE_USER
+        ROLE_SUPER_ADMIN: [ ROLE_ADMIN, ROLE_ALLOWED_TO_SWITCH ]
+
     firewalls:
         dev:
             pattern: ^/(_(profiler|wdt)|css|images|js)/
             security: false
+#
+#        login_firewall:
+#                pattern: ^/saml/metadata
```

Comment:
Dit kan vermoedelijk weg, in principe is je hele applicatie afgeschermd door main/saml_based firewall.